src/applications/check-in/components/pages/validate/ValidateDisplay.jsx

```diff
@@ -121,6 +121,7 @@ export default function ValidateDisplay({
             value={dob}
             required
             onKeyDown={handleEnter}
+            monthSelect={false}
```

Comment:
Hello patient check-in team! 👋 I have added this prop because this component-library release makes the va-memorable-date month input a select by default now (the `month-select` prop defaults to true). I'm adding false here so that your form continues to display it as a text input as it originally was.

Reply:
Thanks @jamigibbs!